napalm/ios/ios.py

```diff
@@ -3376,19 +3376,31 @@ def get_config(self, retrieve="all", full=False):
         since IOS does not support candidate configuration.
         """
 
+        # The output of get_config should be directly usable by load_replace_candidate()
+        # IOS adds some extra, unneeded lines that should be filtered.
+        filter_strings = [r"^Building configuration.*$", r"^Current configuration :.*$"]
```

Comment:
Yeah, let me restructure slight to move the regex filter construction to helpers (as this could be re-used). This is the regex logical-or construction.

Also I wondered if we should filter out things we might not want if you are using `get_config` as part of backup system (revision control system). In other words, Cisco IOS puts timestamps in the config (in certain situations). These are comments so they have no net effect on the actual configuration, but if you want to check them into Git or some other version control system, then it will be detected as a change.

I initially had these filtered, but then I said I would punt that out...as my main objective was to get the configuration retrieved from get_config() to work with a replace operation.